db/migrate/20160318203331_increase_delayed_job_column.rb

```diff
@@ -1,8 +1,11 @@
 class IncreaseDelayedJobColumn < ActiveRecord::Migration[4.2]
   def change
-    # Ensure that the handler column in the delayed_job table is MySQL longtext
-    # so that we can pass in lots of druids in the params when a new Delayed
-    # Job is created.
-    change_column :delayed_jobs, :handler, :text, limit: 4_294_967_295
+    # Only run this migration if MySQL is the database. The :text datatype in Postgres is unlimited.
+    if ActiveRecord::Base.connection.adapter_name.match?(/mysql/i)
```

Comment:
Do we need this if we're configuring postgres and using it in docker? Is this just an "in-case someone is using it" or do we still have mysql out there?

Reply:
@aaron-collier Now that the migration's done in all three envs, I think I'll be inclined to drop this in a follow-up PR. I forgot I even made this branch backwards-compatible. 😆